frontend/src/components/Login/LoginMobile.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import "./login.css";
+
+export default function LoginMobile() {
```

Comment:
Whoops, I forgot to copy the stuff over. Though I'm not entirely sure what practices I should do differently for the mobile version since I have no experience coding for mobile.